extensions/datastores/kudu/src/main/java/org/locationtech/geowave/datastore/kudu/operations/KuduReader.java

```diff
@@ -115,13 +114,10 @@ protected void initDataIndexScanner() {
     }
     try {
       iterator =
-          operations.getKuduRangeRead(
+          (CloseableIterator<T>) operations.getKuduDataIndexRead(
```

Comment:
Is this cast necessary?

Reply:
We can get rid of the cast by calling the method as `operations.<T> getKuduDataIndexRead(`